### PR TITLE
Parallel and reports

### PR DESCRIPTION
**Pull request by Bivis-s**
image

### Added:
- TestNG parallel execution
- Cucumber html reports
- Qase tms integration was added

image

### CODE REVIEW

**Comment by Bivis-s** [comment]
Now there are Allure and screenshots on test failures

screenshot-26 222 46 82_57775-2021 03 19-19_34_40

(the test is specially made failed)
screenshot-26 222 46 82_56471-2021 03 19-19_23_40

File: src/main/java/utils/ScreenshotUtils.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package utils;
+
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+
+public class ScreenshotUtils {
+    private ScreenshotUtils() {
+    }
```
Comment (vchesnakoutest) [inline src/main/java/utils/ScreenshotUtils.java:9]: молодца 👍

File: src/test/java/steps/api/ApiSteps.java [hunk 2]
```diff
@@ -11,26 +11,26 @@
 import property_objects.wrappers.SuitePropertiesWrapper;
 import utils.DateGenerator;
 import utils.RandomStringGenerator;
-import world.World;
+import world_context.WorldContext;
 
 @RequiredArgsConstructor
 public class ApiSteps {
-    private final World world;
+    private final WorldContext worldContext;
```
Comment (vchesnakoutest) [inline src/test/java/steps/api/ApiSteps.java:18]: 👍

**Review by vchesnakoutest — APPROVED** [review]
good job 👍

**Pull request merged**